src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { StoryFn } from '@storybook/react'
+import React from 'react'
+
+import { Stack } from '../Layout'
+
+import { Chip } from '.'
+
+export default {
+  title: 'Data Display（データ表示）/Chip',
+  component: Chip,
+}
+
+export const All: StoryFn = () => (
+  <Stack as="dl" gap={3}>
+    <Stack align="flex-start">
+      <Chip label="ラベル" />
+      <Chip label="Label" />
+      <Chip label="label" />
+      <Chip label="Label ラベル" aria-label="ラベル" />
+    </Stack>
+  </Stack>
```

Comment:
- 謎の `dl` と Stack が残ってそう
- Cluster の方がよく組み合わされそう
- ラベルや Label、label などのバリエーション意図がわからなかった
- `aria-label` もなくてもよさそう

```suggestion
  <Cluster align="center">
    <Chip label="ラベル" />
    <Chip label="Label ラベル" aria-label="ラベル" />
  </Stack>
```

Reply:
`aria-label` は他の属性も渡せるんだよを示したかったためですが、他コンポーネントの基本設計としてできる前提ですかね？　であれば承知しましま。

Reply:
> 他コンポーネントの基本設計としてできる前提です

認識の通りです!
アイコンを渡す場合などに例として追加するとよさそうに思いました。